ansible_runner/interface.py

```diff
@@ -164,6 +164,8 @@ def run(**kwargs):
     :param artifact_dir: The path to the directory where artifacts should live, this defaults to 'artifacts' under the private data dir
     :param project_dir: The path to the playbook content, this defaults to 'project' within the private data dir
     :param rotate_artifacts: Keep at most n artifact directories, disable with a value of 0 which is the default
+    :param timeout: The timeout value in seconds that will be passed to either ``pexpect`` of ``subprocess`` invocation
```

Comment:
This is a good chance, one small tweak to this in that I'd like to explain what the effect of this will be if set... force canceling the execution in this case. I just want to make sure we call that out.

Reply:
Fixed